common/cache/lru_test.go

```diff
@@ -30,9 +30,9 @@ import (
 	"testing"
 	"time"
 
-	"github.com/google/uuid"
-	"github.com/jonboulle/clockwork"
+	"github.com/pborman/uuid"
```

Comment:
I think at some point there was a consensus to move towards the `github.com/google/uuid` package, not away from it